src/components/article/components/ArticleSidebar.vue

```diff
@@ -88,13 +88,13 @@ export default {
         padding: $spacer-0;
     }
 
-    .vs-image-with-caption__image-wrapper {
-        .vs-toggle-btn {
-            @include media-breakpoint-between(sm, md) {
-                display: block;
-            }
-        }
-    }
+    // .vs-article-sidebar__img-wrapper {
+    //     .vs-toggle-btn {
+    //         @include media-breakpoint-between(md, lg) {
+    //             display: block;
+    //         }
+    //     }
+    // }
```

Comment:
If neither of these blocks of code is needed can we get rid of them fully?

Reply:
Yeah so this is a bug I found with the article sidebar image, this code is needed but not until the bug is fixed :D I need to create a separate ticket for the bug, thanks for reminding me